Microsoft.Toolkit.Sample.DesktopWindow/Microsoft.Toolkit.Sample.DesktopWindow.csproj

```diff
@@ -0,0 +1,42 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
+
+  <PropertyGroup>
+    <OutputType>WinExe</OutputType>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <Platforms>x86;x64</Platforms>
+    <!-- Workaround: https://github.com/microsoft/microsoft-ui-xaml/issues/798 -->
+    <TargetPlatformVersion>10.0.18362.0</TargetPlatformVersion>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
+  </PropertyGroup>
```

Comment:
This is missing a `<IsPackable>false</IsPackable>`.
This project is generating a nupkg, and it shouldn't.